test/signin.js

```diff
@@ -0,0 +1,30 @@
+let app = require('../app');
+let chai = require('chai');
+let request = require('supertest')
+let should = chai.should();
+request = request('http://localhost:3000')
```

Comment:
While this works, It's actually really bad practice to overwrite a reference with a reference of the same name.

I would recommend just doing the following if you don't want an extra reference:

```
let request = require('supertest')('http://localhost:3000');
```